src/styles/styles.js

```diff
@@ -632,11 +638,20 @@ const styles = {
         borderRadius: variables.componentBorderRadiusNormal,
         zIndex: -1,
     },
+
     expensiTextInputDesktop: addOutlineWidth({}, 0),
+
     expensiTextInputAndroid: left => ({
         padding: 0,
         left,
     }),
+
+    secureInputEyeButton: {
+        paddingRight: 11.5,
+        marginLeft: -3.5,
```

Comment:
Well, this margin seems off. Why do we need it?

Reply:
it's to reduce the distance between the icon and the input element to be 8px.
I decided to do it with a negative margin so I didn't have to add any JS logic to the stylings based of if the icon was there or not. I generally prefer to do things in CSS if I can without adding JS.

Reply:
Well, the negative margin has side-effect on Android and thus we should not use it on the Clickable element. https://reactnative.dev/docs/handling-touches#known-issues

Anyways, the whole app is built on js so we should not worry about using a flag to adjust the style.

Reply:
can't argue with not supported!